pagopa/pn-frontend
# Pn 1789 fe configurare sonar correttamente per eseguire correttamente la coverage analysis

## Short description
Added correct sonar properties file config for each package

## List of changes proposed in this pull request
- improved performance during install phase in CI
- sonar.scm.exclusions.disabled changed to true in order to start coverage analysis in sonar projects
- configured exclusion paths for coverage analysis

## How to test
Go to sonar dashboard for related projects and check that for this PR coverage analysis is computed.
https://sonarcloud.io/project/overview?id=pagopa_pn-pa-webapp 
https://sonarcloud.io/project/overview?id=pagopa_pn-personafisica-login
https://sonarcloud.io/project/overview?id=pagopa_pn-personafisica-webapp
https://sonarcloud.io/project/overview?id=pagopa_pn-validator
https://sonarcloud.io/project/overview?id=pagopa_pn-fe-commons
